packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -114,7 +112,11 @@ export class DumbTriggerManager extends Component {
     try {
       this.setState({ error: null, status: RUNNING })
       const oAuthAccount = await findAccount(client, accountId)
-      return await this.handleNewAccount(oAuthAccount)
+      return await flow.ensureTriggerAndLaunch(client, {
```

Comment:
Were does `flow` come from? Shouldn't it be extracted from `this.props`?
(Maybe fixed later, currently reviewing https://github.com/cozy/cozy-libs/pull/956/commits/f35bfa35574945ea2412d663165b13cc0fea077f)